src/Components/Search/Suggestions/SuggestionItem.tsx

```diff
@@ -110,15 +114,35 @@ const DefaultSuggestion: React.FC<SuggestionItemProps> = ({
   )
 
   return (
-    <>
-      <Text variant="sm" overflowEllipsis>
-        {partTags}
-      </Text>
-
-      <Text color="black60" variant="xs" overflowEllipsis>
-        {label}
-      </Text>
-    </>
+    <Flex>
+      <Box width={50} height={50} mr={1}>
+        {imageUrl ? (
+          <Image
+            lazyLoad
+            width={50}
+            height={50}
+            src={imageUrl}
+            borderRadius={label === "Artist" ? 50 : 0}
+          />
+        ) : (
+          <Box
+            width={50}
+            height={50}
+            bg="black5"
+            borderRadius={label === "Artist" ? 50 : 0}
+          />
+        )}
+      </Box>
```

Comment:
**suggestion**: create a separate variable with preview image size. something like this

```javascript
const PREVIEW_IMAGE_SIZE = 50

...

<Box width={PREVIEW_IMAGE_SIZE} height={PREVIEW_IMAGE_SIZE} mr={1}>
  {imageUrl ? (
    <Image
      lazyLoad
      width={PREVIEW_IMAGE_SIZE}
      height={PREVIEW_IMAGE_SIZE}
      src={imageUrl}
      borderRadius={label === "Artist" ? PREVIEW_IMAGE_SIZE : 0}
    />
  ) : (
    <Box
      width={PREVIEW_IMAGE_SIZE}
      height={PREVIEW_IMAGE_SIZE}
      bg="black5"
      borderRadius={label === "Artist" ? PREVIEW_IMAGE_SIZE : 0}
    />
  )}
</Box>
```

Reply:
**suggestion**: I also think it would be a good idea to move it in a separate component. Something like this

```javascript
const PREVIEW_IMAGE_SIZE = 50

interface SuggestionItemPreviewProps {
  imageUrl?: string
  label: string
}

const SuggestionItemPreview: React.FC<SuggestionItemPreviewProps> = ({
  imageUrl,
  label,
}) => {
  const isArtistLabel = label === "Artist"
  const borderRadius = isArtistLabel ? 50 : 0

  if (imageUrl) {
    return (
      <Image
        lazyLoad
        width={PREVIEW_IMAGE_SIZE}
        height={PREVIEW_IMAGE_SIZE}
        src={imageUrl}
        borderRadius={borderRadius}
      />
    )
  }

  return (
    <Box
      width={PREVIEW_IMAGE_SIZE}
      height={PREVIEW_IMAGE_SIZE}
      bg="black5"
      borderRadius={borderRadius}
    />
  )
}


...

<SuggestionItemPreview imageUrl={imageUrl} label={label} />

<Separator x={1} />

<Flex flexDirection="column" mr={1} flex={1} overflow="hidden">
  <Text variant="sm-display" overflowEllipsis>
    {partTags}
  </Text>

  <Text color="black60" variant="xs" overflowEllipsis>
    {label}
  </Text>
</Flex>
```